kits/agent-failure-investigator/prompts/compose-root-cause_narrate-root-cause_user.md

```
@@ -0,0 +1,8 @@
+Primary failure category: {{triggerNode_1.output.primaryCategory}} (confidence {{triggerNode_1.output.confidence}}%)
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)